test/remote/db_tests.js

```diff
@@ -184,25 +196,47 @@ describe('remote db', function() {
           assert.equal(sessionToken.emailCode, account.emailCode)
           assert.equal(sessionToken.emailVerified, account.emailVerified)
           assert.equal(sessionToken.lifetime < Infinity, true)
-          return sessionToken
-        })
-        .then(function(sessionToken) {
-          // override lastAccessTimeUpdates flag
+
+          // Disable session token updates
           lastAccessTimeUpdates.enabled = false
+
+          // Attempt to update the session token
           return db.updateSessionToken(sessionToken)
         })
-        .then(function(result) {
-          assert.equal(undefined, result)
+        .then(result => {
+          assert.equal(result, undefined)
```

Comment:
Swapped these so that `actual` and `expected` are the right way round in the error message.